examples/gno.land/r/stefann/home/home.gno

```diff
@@ -0,0 +1,274 @@
+package home
+
+import (
+	"sort"
+	"std"
+
+	"gno.land/p/demo/ufmt"
+
+	"gno.land/r/stefann/config"
+)
+
+type City struct {
+	Name string
+	URL  string
+}
+
+type Sponsor struct {
+	Address std.Address
+	Amount  std.Coins
+}
+
+var (
+	pfp              string
+	cities           []City
+	currentCityIndex int
+	aboutMe          [2]string
+	jarLink          string
+	maxSponsors      int
+	sponsors         []Sponsor
+	totalDonated     std.Coins
+	totalDonations   int
+)
+
+func init() {
+	pfp = "https://i.ibb.co/Bc5YNCx/DSC-0095a.jpg"
+	cities = []City{
+		{Name: "Venice", URL: "https://i.ibb.co/1mcZ7b1/venice.jpg"},
+		{Name: "Tokyo", URL: "https://i.ibb.co/wNDJv3H/tokyo.jpg"},
+		{Name: "São Paulo", URL: "https://i.ibb.co/yWMq2Sn/sao-paulo.jpg"},
+		{Name: "Toronto", URL: "https://i.ibb.co/pb95HJB/toronto.jpg"},
+		{Name: "Bangkok", URL: "https://i.ibb.co/pQy3w2g/bangkok.jpg"},
+		{Name: "New York", URL: "https://i.ibb.co/6JWLm0h/new-york.jpg"},
+		{Name: "Paris", URL: "https://i.ibb.co/q9vf6Hs/paris.jpg"},
+		{Name: "Kandersteg", URL: "https://i.ibb.co/60DzywD/kandersteg.jpg"},
+		{Name: "Rothenburg", URL: "https://i.ibb.co/cr8d2rQ/rothenburg.jpg"},
+		{Name: "Capetown", URL: "https://i.ibb.co/bPGn0v3/capetown.jpg"},
+		{Name: "Sydney", URL: "https://i.ibb.co/TBNzqfy/sydney.jpg"},
+		{Name: "Oeschinen Lake", URL: "https://i.ibb.co/QJQwp2y/oeschinen-lake.jpg"},
+		{Name: "Barra Grande", URL: "https://i.ibb.co/z4RXKc1/barra-grande.jpg"},
+		{Name: "London", URL: "https://i.ibb.co/CPGtvgr/london.jpg"},
+	}
+	currentCityIndex = 0
+	jarLink = "https://TODO"
+	maxSponsors = 5
+	aboutMe = [2]string{
+		`<h3 style="font-size: 1.4em;">About Me</h3>
+        <p style="font-size: 1.1em;">Hey there! I’m Stefan, a student of Computer Science. I’m all about exploring and adventure — whether it’s diving into the latest tech or discovering a new city, I’m always up for the challenge!</p>`,
+		`<h3 style="font-size: 1.4em;">Contributions</h3>
+        <p style="font-size: 1.1em;">I'm just getting started, but you can follow my journey through Gno.land right here <a href="https://github.com/gnolang/hackerspace/issues/94" target="_blank">🔗</a></p>`,
+	}
+}
+
+func UpdateMaxSponsors(newMax int) {
+	config.AssertAuthorized()
+	maxSponsors = newMax
+}
+
+func UpdateCities(newCities []City) {
+	config.AssertAuthorized()
+	cities = newCities
+}
+
+func UpdateJarLink(newLink string) {
+	config.AssertAuthorized()
+	jarLink = newLink
+}
+
+func UpdatePFP(url, caption string) {
+	config.AssertAuthorized()
+	pfp = url
+}
+
+func UpdateAboutMe(col1, col2 string) {
```

Comment:
Fixed! (commit: f37732a2b70ba8f2431c926c428206e27f1434e7)

Switched to `md` instead of `html` to better align with future plans for `gnoweb`.
Faced issues with separating columns (rows actually, my bad) and rendering new lines due to syntax limitations in `gnokey maketx` arguments. Resolved this by introducing a special character (`|`) to separate rows and placing titles in individual rows rather than relying on new lines.